src/Core/src/Platform/Android/EditTextExtensions.cs

```diff
@@ -41,10 +42,18 @@ public static void UpdateTextColor(this EditText editText, ITextStyle entry)
 
 		public static void UpdateTextColor(this EditText editText, Graphics.Color textColor)
 		{
-			if (textColor != null)
+			if (textColor is not null && PlatformInterop.CreateEditTextColorStateList(editText.TextColors, textColor.ToPlatform()) is ColorStateList c)
+				editText.SetTextColor(c);
+			else
 			{
-				if (PlatformInterop.CreateEditTextColorStateList(editText.TextColors, textColor.ToPlatform()) is ColorStateList c)
-					editText.SetTextColor(c);
+				var typedValue = new TypedValue();
```

Comment:
Can you include a couple of comments here? Like, `// Fallback to system default color`